lib/spec_test.go

```diff
@@ -10,10 +11,21 @@ import (
 	"github.com/zclconf/go-cty/cty"
 )
 
+func TestReadSpecMetadataMock(t *testing.T) {
+	spec, diags := ReadSpecMetadataMock("testdata/scenario.tfspec")
+	if diags.HasErrors() {
+		t.Fatal(diags.Err())
+	}
+
+	if spec == nil {
+		t.Fatal("spec is nil")
+	}
+}
+
```

Comment:
This test function doesn't test the most important. Does it correctly read the mockMetadata set in the tfspec file ?

Reply:
Yup it does. Totally forgot to add the test here. I manually tested end2end